deploy/roles/r-studio-server-config/tasks/main.yml

```diff
@@ -48,6 +51,7 @@
   template:
     src: rsession.conf.j2
     dest: /etc/rstudio/rsession.conf
+  when: update_rstudio
```

Comment:
@dvbalen I think we only want the `when: update_rstudio` conditional for things that should only happen when R Studio is being upgraded to a newer version, no?  We may want to update the configuration files when not upgrading from time to time, to change settings.